lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -234,12 +234,11 @@ class ReportUIFeatures {
     tablesWithUrls.forEach((tableEl, index) => {
       const urlItems = this._getUrlItems(tableEl);
       const thirdPartyRows = this._getThirdPartyRows(tableEl, urlItems, this.json.finalUrl);
-      // If all or none of the rows are 3rd party, no checkbox!
-      if (thirdPartyRows.size === urlItems.length || !thirdPartyRows.size) return;
 
       // create input box
       const filterTemplate = this._dom.cloneTemplate('#tmpl-lh-3p-filter', this._templateContext);
-      const filterInput = this._dom.find('input', filterTemplate);
+      const filterInput =
+        /** @type {HTMLInputElement} */ (this._dom.find('input', filterTemplate));
```

Comment:
man it'd really be nice to type this for some basic `HTMLXElement` inference

Reply:
but how often do we even select on just the tagname?

Reply:
not enough to make me want to put in the effort to do it for real :)

![image](https://user-images.githubusercontent.com/2301202/60373768-0ac90a80-99c7-11e9-9ebc-2c067a826bb4.png)